os/fs/smartfs/smartfs_smart.c

```diff
@@ -1986,9 +1961,8 @@ int smartfs_rename(struct inode *mountpt, const char *oldrelpath, const char *ne
 			goto errout_with_semaphore;
 		}
 
-		/* Nope, it's a directory.  Now search the directory for oldfilename */
+		/* Nope, it's a directory.  Now search the directory for old file name */
 
-		newfilename = oldfilename;
```

Comment:
oldentry.dsector is correct?
```
fdbg("Error %d reading sector %d data\n", ret, oldentry.dsector);
```

Reply:
Hmm, I think we lost consistency here with multiple changes.
I'll take care of this.